### PR TITLE
Footer enhancements

### PR DESCRIPTION
**Pull request by nirmalkc**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

Current footer is quite simple. Need more additional information about the vision of the project, stack used, purpose, contact options etc.,

# Fixes given

- Footer logo
- A short description about the owner
- Social media connecting options
- Description about the framework
- What can you do using the framework
- How can you start with this framework
- How can you show your support to this framework by a tweet or being a sponsor.

### CODE REVIEW

**Comment by nirmalkc** [comment]
@atapas , need your help in the following areas to get this merged.

1. Share link integration with the modal.
2. Sponsor link integration
3. ID review for all the text

File: src/common/techstack/techstack.jsx
Comment (atapas) [inline src/common/techstack/techstack.jsx:1]: The file name is not meeting the convention

**Review by atapas — CHANGES_REQUESTED** [review]
Also check out the new warnings and fix.

File: src/common/techstack/techstack.jsx
Comment (atapas) [inline src/common/techstack/techstack.jsx:2]: @Angryman18 My comment was about this file name.

**Review by atapas — CHANGES_REQUESTED** [review]

File: src/common/techstack/techstack.jsx
Comment (Angryman18) [reply src/common/techstack/techstack.jsx:2]: i guess it is done.

**Review by Angryman18 — APPROVED** [review]

File: src/common/techstack/TechStackInfo.js
Comment (koustov) [inline src/common/techstack/TechStackInfo.js:95]: time to bring python,node etc.

**Review by koustov — CHANGES_REQUESTED** [review]
dropped few comments

File: src/common/techstack/TechStackInfo.js
Comment (atapas) [reply src/common/techstack/TechStackInfo.js:95]: Yeah we should. Do we use plop anymore after your PR @koustov ? If not make sure to remove from package.json on your branch.

@Angryman18 this change is on you

File: src/common/header/HeaderNav.jsx
Comment (atapas) [reply src/common/header/HeaderNav.jsx:14]: @Angryman18 this change is on you

File: src/common/footer/Footer.jsx
Comment (atapas) [reply src/common/footer/Footer.jsx:189]: @nirmalkc is fixing the size.

File: src/common/footer/Footer.jsx
Comment (atapas) [reply src/common/footer/Footer.jsx:163]: As we are making it very prominent in the first fold already, can omit here.

File: src/common/footer/Footer.jsx
Comment (atapas) [reply src/common/footer/Footer.jsx:124]: Share

@Angryman18 this change is on you

File: src/common/footer/Footer.jsx
Comment (atapas) [reply src/common/footer/Footer.jsx:123]: lean

@Angryman18 this change is on you

File: src/App.css
Comment (atapas) [reply src/App.css:235]: Not in this PR.

File: src/App.css
Comment (atapas) [reply src/App.css:211]: @nirmalkc to take care

File: src/common/footer/Footer.jsx
Comment (atapas) [reply src/common/footer/Footer.jsx:22]: @Angryman18 Can you please respond?

File: src/common/header/HeaderNav.jsx
Comment (Angryman18) [reply src/common/header/HeaderNav.jsx:14]: this style is for sx in Box element from Mui. i not sure how to pass it to sx after importing from css file as className?

File: src/common/techstack/TechStackInfo.js
Comment (Angryman18) [reply src/common/techstack/TechStackInfo.js:97]: added

File: src/common/footer/Footer.jsx
Comment (Angryman18) [reply src/common/footer/Footer.jsx]: yes typo fixed

File: src/common/footer/Footer.jsx
Comment (Angryman18) [reply src/common/footer/Footer.jsx]: fixed

File: src/common/footer/Footer.jsx
Comment (Angryman18) [reply src/common/footer/Footer.jsx]: fixed

Comment (Angryman18) [reply src/common/footer/Footer.jsx]: created a new component extended footer and added to its path designated component.

File: src/App.css
Comment (koustov) [reply src/App.css:235]: I think its connected. Otherwise, it will always be "Not in this PR"

Comment (nirmalkc) [reply src/App.css:243]: Moving to SCSS is a big step. Let's do that in a separate PR. In this PR, my recommendation would be to close on the footer and header supporting share/sponsor actions.

File: src/common/footer/Footer.jsx
Comment (nirmalkc) [reply src/common/footer/Footer.jsx:10]: taken care

**Comment by atapas** [comment]
@koustov could you please finish the review?

File: src/common/footer/Footer.jsx
Comment (koustov) [reply src/common/footer/Footer.jsx:10]: What about ` I feel it deserves a dedicated section on the home page`?

Comment (atapas) [reply src/common/footer/Footer.jsx:10]: @nirmalkc what do you think? Can you please respond?

File: src/common/header/HeaderNav.jsx
Comment (atapas) [reply src/common/header/HeaderNav.jsx:14]: @Angryman18 @koustov could you both please come to a point on this?

Comment (Angryman18) [reply src/common/header/HeaderNav.jsx:14]: I have created a new Modal component (from Mui modal) currently while building the authentication flow and it contains these stylese and with different sizes so in future we will replace this modal after we merge authentication branch. looking forward to see other thoughts also.

**Review by koustov — APPROVED** [review]
Approving with conditions
1. @nirmalkc please create issue to convert the complex CSS files to SCSS
2. Create another issue if you feel that we should render a small section for `tech stack` (maybe a categorized one).

**Comment by atapas** [comment]
> Approving with conditions
> 
> 1. @nirmalkc please create issue to convert the complex CSS files to SCSS
> 2. Create another issue if you feel that we should render a small section for `tech stack` (maybe a categorized one).



> Approving with conditions
> 
> 1. @nirmalkc please create issue to convert the complex CSS files to SCSS
> 2. Create another issue if you feel that we should render a small section for `tech stack` (maybe a categorized one).

I have created an issue for the first one: https://github.com/reactplay/react-play/issues/374

I'll let @nirmalkc decide on the second. Going ahead with merge.

**Review by atapas — APPROVED** [review]

**Comment by nirmalkc** [comment]
> > Approving with conditions
> > 
> > 1. @nirmalkc please create issue to convert the complex CSS files to SCSS
> > 2. Create another issue if you feel that we should render a small section for `tech stack` (maybe a categorized one).
> 
> > Approving with conditions
> > 
> > 1. @nirmalkc please create issue to convert the complex CSS files to SCSS
> > 2. Create another issue if you feel that we should render a small section for `tech stack` (maybe a categorized one).
> 
> I have created an issue for the first one: #374
> 
> I'll let @nirmalkc decide on the second. Going ahead with merge.

I have created a new issue for the tech stack page.
https://github.com/reactplay/react-play/issues/375

**Review by atapas — APPROVED** [review]

**Pull request merged**
